### DISCUSSION TITLE
Multiply censored data

### DISCUSSION THREAD

**Discussion by Madeos** [tvtoglu/predictr#6] [General]
Hi,

I am new to Weibull analysis and enjoy the acessibility of this package! Literature often distinguishes between singly, multiple and interval censored data. How is this handled in the package? Is there anything to take care when using differently censored data?

Cheers,
Lukas

**Comment by tvtoglu** [comment]
Hi Lukas,

sorry for the late reply!

predictr can handle singly and multiply right-censored data (Type I and type II right-censored). You don't need to set anything, predictr already handles the data automatically the correct way.

There is no implementation for interval censored data yet. I might add this feature in the future.

Feel free to ask me more questions, I am happy to answer them. 

Best regards,
Tamer
